tests/test_wfirst.py

```diff
@@ -536,15 +536,31 @@ def test_wfirst_detectors():
     im_list = [im0_1,im0_2]*4
 
     im_1.applyPersistence(im_list,galsim.wfirst.persistence_coefficients)
-    galsim.wfirst.applyPersistence(im_2, im_list)
+    galsim.wfirst.applyPersistence(im_2, im_list, method='linear') #check the linear method
     assert im_2.scale == im_1.scale
     assert im_2.wcs == im_1.wcs
     assert im_2.dtype == im_1.dtype
     assert im_2.bounds == im_1.bounds
     np.testing.assert_array_equal(
         im_2.array, im_1.array,
         err_msg='Persistence results depend on function used.')
+
+    im_unit = galsim.Image(np.ones((2,2)), copy=True)
+    im_f = im_unit*0.0
+    im_f1 = im_unit*0.0
+    illuminatin_list = [1.E3, 1.E4, 4.E4, 4.99E4, 5.01E4, 1.E5, 1.0E6]
+    im_f_list = [x*im_unit for x in illuminatin_list]
+
+    galsim.wfirst.applyPersistence(im_f, im_f_list, method='fermi') #check fermi method
+    galsim.wfirst.applyPersistence(im_f1, im_f_list)
+
+    #Check the functionality of the fermi method by comparing with pre-calculated values.
```

Comment:
The comment should say what is the origin of the pre-calculated values, because if the routine changes (for example due to updated coefficients) then whoever changes it needs to know how to precalculate values to update the unit test.

Reply:
Even better would be to do the calculations here in the test suite rather than externally.  With inline comments about what formula you are using to do so and where that formula comes from.

Reply:
I  added the calculation in the test function. No hard coded value is needed.